### ISSUE TITLE
🐛 

### ISSUE THREAD

**Issue by SimonShvets** [AutoFixture/AutoFixture#1397]
### Screenshots

<img width="277" alt="image" src="https://github.com/AutoFixture/AutoFixture/assets/74120573/0a467f2b-b2f4-4bfa-bb07-de4bd307e4e1">
<img width="187" alt="image" src="https://github.com/AutoFixture/AutoFixture/assets/74120573/8a4a8b06-d932-4034-b308-ed91dfcc4854">

Thanks!

**Comment by aivascu** [comment]
@SimonShvets yes this is a known bug. I've been working on the fix for a while now as it required some internal redesign of the xUnit integration. The fix should come with the next major version, some time soon.

**Comment by aivascu** [comment]
Yes all the supported data sources from xUnit 2 are already dev complete.

**Comment by aivascu** [comment]
Duplicate of #1142
